backend/internal/service/handler/media/get_media_by_name.go

```diff
@@ -102,6 +142,14 @@ func (h *Handler) handleSearchResults(client *spotify.Client, ctx context.Contex
 
 	}
 
+	for _, artist := range result.Artists.Artists {
```

Comment:
i think we should wrap this for loop with this so that it doesn't break for the existing search endpoint

```
if result.Artists != nil {

}
```